packages/shared/src/variants/parallel.ts

```diff
@@ -31,9 +31,8 @@ export class ParallelGo extends AbstractGame<
   private board: Grid<number[]>;
   private staged: MovesType = {};
   private last_round: MovesType = {};
-  private playerParticipation = this.initializeParticipation(
-    this.config.num_players,
-  );
+  // Resignations and timeouts
+  private dropouts = new Set<number>();
```

Comment:
I noticed we aren't doing much with the playerParticipation data (i.e. round), so I decided to replace it with "dropouts" for easier access